libcudacxx/include/cuda/std/detail/libcxx/include/bitset

```diff
@@ -130,931 +165,848 @@ template <size_t N> struct hash<std::bitset<N>>;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 void operator^=(const __bitset& __v) noexcept;
+
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 void flip() noexcept;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long to_ulong() const {
+    return to_ulong(integral_constant < bool, _Size< sizeof(unsigned long) * CHAR_BIT>());
+  }
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long long to_ullong() const {
+    return to_ullong(integral_constant < bool, _Size< sizeof(unsigned long long) * CHAR_BIT>());
+  }
+
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 bool all() const noexcept;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 bool any() const noexcept;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY size_t __hash_code() const noexcept;
+
 private:
-    unsigned long to_ulong(false_type) const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    unsigned long to_ulong(true_type) const;
-    unsigned long long to_ullong(false_type) const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    unsigned long long to_ullong(true_type) const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    unsigned long long to_ullong(true_type, false_type) const;
-    unsigned long long to_ullong(true_type, true_type) const;
+#ifdef _LIBCUDACXX_CXX03_LANG
+  void __init(unsigned long long __v, false_type) noexcept;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY void __init(unsigned long long __v, true_type) noexcept;
+#endif // _LIBCUDACXX_CXX03_LANG
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long to_ulong(false_type) const;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long to_ulong(true_type) const;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long long to_ullong(false_type) const;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long long to_ullong(true_type) const;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long long to_ullong(true_type, false_type) const;
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 unsigned long long to_ullong(true_type, true_type) const;
 };
 
 template <size_t _N_words, size_t _Size>
-inline constexpr
-__bitset<_N_words, _Size>::__bitset() noexcept
+inline constexpr __bitset<_N_words, _Size>::__bitset() noexcept
+#ifndef _LIBCUDACXX_CXX03_LANG
     : __first_{0}
-{}
+#endif
+{
+#ifdef _LIBCUDACXX_CXX03_LANG
+  std::fill_n(__first_, _N_words, __storage_type(0));
+#endif
+}
```

Comment:
We recently droppped all usage of `_LIBCUDACXX_CXX03_LANG` as that is always undefined
Also can we please inline all the function definitions into the class body. 

We do not have any source files, so it only complicates readability